### PR TITLE
Enable new partitioner UI flow on TW staging

### PR DESCRIPTION
**Pull request by rwx788**
See [poo#42128](https://progress.opensuse.org/issues/42128).

* [needles](https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/458)
* [verification run](http://g226.suse.de/tests/2877)

### CODE REVIEW

File: lib/partition_setup.pm [hunk 1]
```diff
@@ -40,7 +40,7 @@ our %partition_roles = qw(
 
 # We got changes to the storage-ng UI in SLE 15 SP1, Leap 15.1 and TW
 sub is_storage_ng_newui {
-    return is_storage_ng && (is_sle('15-SP1+') || is_leap('15.1+'));
+    return is_storage_ng && (is_sle('15-SP1+') || is_leap('15.1+') || is_tumbleweed && is_staging);
```
Comment (okurz) [inline lib/partition_setup.pm]: This will break other staging projects. Can you make the detection dynamic? If not then check for :H specifically

Comment (rwx788) [reply lib/partition_setup.pm]: I will add variable to do same as we did for system roles.

Comment (rwx788) [reply lib/partition_setup.pm]: Dynamic detection is a hacky solution as it means that we don't know what we expect. Just variable won't work as we have single medium for all staging projects, so I enabled it for H specifically and that we can enable it everywhere without urgent code change. I've added this topic to the retro, the process doesn't look good to me.

**Comment by okurz** [comment]
Needles are not yet merged but as the changes should only have an effect on :H I guess we can go ahead

**Pull request merged**